docs_src/pipelinebenchmarking.md

```diff
@@ -1,7 +1,10 @@
 # Pipeline Benchmarking
 
+Pipeline benchmarking are done through collections of scripts to obtain the metrics of pipeline performance like video processing in frame-per-second (FPS),
```

Comment:
Slight wording suggestion

Pipeline benchmarking is done through a collection of scripts to...